.github/workflows/docker.yml

```diff
@@ -21,12 +21,46 @@
 # SOFTWARE.
 ---
 name: docker
-'on':
+on:
   push:
   pull_request:
+
 jobs:
   docker:
     runs-on: ubuntu-22.04
     steps:
       - uses: actions/checkout@v4
-      - run: docker build . -t home
+      - uses: docker/setup-qemu-action@v2
+      - uses: docker/setup-buildx-action@v2
+
+      - name: Docker meta
+        id: meta
+        uses: docker/metadata-action@v5
+        with:
+          images: |
+            yegor256/squid-proxy
+          # yamllint disable rule:line-length
+          tags: |
+            type=raw,value=latest,enable=${{ github.ref == format('refs/heads/{0}', 'master') }}
+            type=ref,event=branch
+            type=ref,event=pr
+            type=semver,pattern={{version}}
+            type=semver,pattern={{major}}.{{minor}}
+            type=sha
```

Comment:
Agh, I'm not really sure how to do that. Maybe we should just remove them for now since we have the `latest` and sha tag to start with? And then one of us can figure out how to get proper semver in place?

Reply:
@iloveitaly sounds right, just `latest` is enough